lte/gateway/deploy/roles/agw_docker/tasks/main.yml

```diff
@@ -63,7 +63,7 @@
   template:
     src: env.j2
     dest: /var/opt/magma/docker/.env
-    mode: 0400
+    mode: 0644
```

Comment:
this contains credentials, so it should just be readable for the one that needs them.
It is generated, so it should just be readable, and not writable.

Reply:
So if I revert this back to `0400` then I update the docs with `sudo docker-compose` commands?

Reply:
yes, probably is better

Reply:
Okay, I will do these changes.